### PR TITLE
refactor(components.sectionbacklink): move to telecomUniverseComponents

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

80306bd1 - refactor(components.sectionbacklink): update attributes
891c8102 - refactor(components.sectionbacklink): move to telecomUniverseComponents

ref: OM-111

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

File: client/telecomUniverseComponents/section-back-link/section-back-link.component.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (FredericEspiau) [inline client/telecomUniverseComponents/section-back-link/section-back-link.component.js]: Whitespace

File: client/app/telecom/telephony/alias/configuration/timeCondition/easyHunting/telecom-telephony-alias-configuration-time-condition-easy-hunting.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <section class="telecom-telephony-alias-configuration-timeCondition-easy-hunting">
 
     <header>
-        <section-back-link data-section-back-link-to-state="telecom.telephony.alias.configuration"
+        <tuc-section-back-link data-section-back-link-to-state="telecom.telephony.alias.configuration"
```
Comment (marie-j) [inline client/app/telecom/telephony/alias/configuration/timeCondition/easyHunting/telecom-telephony-alias-configuration-time-condition-easy-hunting.html]: Attributes were not prefixed here ?
```suggestion
        <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.alias.configuration"
```

File: client/app/telecom/telephony/alias/configuration/timeCondition/oldPabx/telecom-telephony-alias-configuration-time-condition-old-pabx.html [hunk 3]
```diff
@@ -1,9 +1,9 @@
 <section class="telecom-telephony-alias-configuration-timeCondition-easy-hunting">
 
     <header>
-        <section-back-link data-section-back-link-to-state="telecom.telephony.alias.configuration"
+        <tuc-section-back-link data-section-back-link-to-state="telecom.telephony.alias.configuration"
```
Comment (marie-j) [inline client/app/telecom/telephony/alias/configuration/timeCondition/oldPabx/telecom-telephony-alias-configuration-time-condition-old-pabx.html]: Same here 
```suggestion
        <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.alias.configuration"
```

File: client/app/telecom/telephony/line/calls/timeCondition/telecom-telephony-line-calls-time-condition.html [hunk 4]
```diff
@@ -1,9 +1,9 @@
 <section class="telecom-telephony-line-calls-timeConditions">
 
     <header>
-        <section-back-link data-section-back-link-to-state="telecom.telephony.line.calls"
+        <tuc-section-back-link data-section-back-link-to-state="telecom.telephony.line.calls"
```
Comment (marie-j) [inline client/app/telecom/telephony/line/calls/timeCondition/telecom-telephony-line-calls-time-condition.html]: Same here, is `scriptinette` broken ? 😄 
```suggestion
        <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.calls"
```

File: client/app/telecom/telephony/alias/configuration/timeCondition/easyHunting/telecom-telephony-alias-configuration-time-condition-easy-hunting.html [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
                            data-section-back-link-title="{{ 'telephony_alias_configuration_back_link' | translate }}">
```
Comment (marie-j) [inline client/app/telecom/telephony/alias/configuration/timeCondition/easyHunting/telecom-telephony-alias-configuration-time-condition-easy-hunting.html]:
```suggestion
                           data-tuc-section-back-link-title="{{ 'telephony_alias_configuration_back_link' | translate }}">
```

File: client/app/telecom/telephony/line/calls/timeCondition/telecom-telephony-line-calls-time-condition.html (hunk 4 above)
Comment (FredericEspiau) [reply client/app/telecom/telephony/line/calls/timeCondition/telecom-telephony-line-calls-time-condition.html]: Should have used AST

File: client/telecomUniverseComponents/section-back-link/index.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import angular from 'angular';
+
+import tucSectionBackLink from './section-back-link.component';
+
+export default angular
+  .module('tucSectionBackLink', [])
```
Comment (antleblanc) [inline client/telecomUniverseComponents/section-back-link/index.js]: Missing `ui.router`?
ref: https://github.com/ovh-ux/ovh-manager-telecom/blob/80cb5d72447eab46a5bc9ddea6313a853e882e82/client/telecomUniverseComponents/section-back-link/section-back-link.html#L12

Comment (antleblanc) [reply client/telecomUniverseComponents/section-back-link/index.js]: I've added the missing `ui.router` dependency.

**Pull request merged**